tests/solana.rs

```diff
@@ -1438,26 +1366,29 @@ impl<'a> SyscallObject<UserError> for SyscallInvokeSignedC<'a> {
                 vm.stack.insert(0, p);
 
                 let res = vm.execute(&instruction.accounts, &instruction.data);
-                assert_eq!(res, Ok(0));
+                assert!(matches!(res, ProgramResult::Ok(0)));
```

Comment:
Nitpick: uses one less macro and makes it less verbose.

```suggestion
                assert_eq!(res.unwrap(), 0);
```

Reply:
This is good point, `unwrap()` is available on `ProgramResult` so I've updated these.